src/serverSetup/sentry.js

```diff
@@ -0,0 +1,17 @@
+import * as Sentry from '@sentry/node'
+import { nodeProfilingIntegration } from '@sentry/profiling-node'
+
+const setupSentry = (app) => {
+  if (process.env.SENTRY_ENABLED?.toLowerCase() === 'true') {
+    Sentry.init({
+      dsn: process.env.SENTRY_DSN,
+      integrations: [nodeProfilingIntegration()],
+      tracesSampleRate: parseFloat(process.env.SENTRY_TRACING_SAMPLE_RATE || '0.01'),
+      profilesSampleRate: parseFloat(process.env.SENTRY_PROFILES_SAMPLE_RATE || '0.01')
+    })
```

Comment:
Let's also set debug option via an environment variable

Reply:
Wait - isn't it necessary to read the value of the SENTRY_DEBUG environment variable?  What if someone had set the variable with the string value "false"?

Reply:
ok do you mean check if it is boolean? Will do this change now